examples/interaction/tooltips/tooltip_content.py

```diff
@@ -1,3 +1,12 @@
+''' A visualization of inputs through Tooltip in bokeh.models.
+This example demonstrates defining two distinct tooltip renderers,
+i.e. plaintext and html tooltip, and accepting text input through those renderers.
```

Comment:
This is a bit roundabout. The primary motivation for the example is to demonstrate how tooltips can be added to widgets (not renderers) and what kinds of content that can be. Everything else (e.g. the fact that they are input widgets) is really secondary to that, and not important to emphasize.